Doc/library/array.rst

```diff
@@ -62,6 +62,14 @@ The actual representation of values is determined by the machine architecture
 (strictly speaking, by the C implementation).  The actual size can be accessed
 through the :attr:`itemsize` attribute.
 
+The module defines the following items:
```

Comment:
Why the plural "items" here, yet it is just one? And yet the type is correctly singularized. I think we can be consistent, the singular wording looks more appropriate.

Reply:
Yes, good catch! 👍